rtengine/dcrop.cc

```diff
@@ -1128,53 +1128,17 @@ void Crop::update(int todo)
     }
 
     if (todo & M_RGBCURVE) {
-        Imagefloat *workingCrop = baseCrop;
```

Comment:
Please see https://github.com/Beep6581/RawTherapee/pull/5934. I don't think removing this is wise atm.

Reply:
Ok, I will revert the change to dcrop.cc